resources/cmake/ConnextDdsAddSecurityArtifacts.cmake

```diff
@@ -0,0 +1,186 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+#[[.rst:
+.. _connextdds_add_security_artifacts:
+
+ConnextDdsAddSecurityArtifacts
+------------------------------
+
+Generate the artifacts needed for secure examples.
+
+``connextdds_add_security_artifacts()``
+
+It is based on:
+ - The OpenSSL configuration files in the resources/security/<ca> directory.
+
+ - The Governance and Permissions files in the resources/xml directory.
+
+So far this means:
+
+ - One ECDSA CA and two peers.
+
+ - RTPS and metadata protection kind set to ENCRYPT.
+
+ - Publishing and subscribing to any domain and topic is allowed, only for those
+   peers.
+
+#]]
+
+include(UseOpenSSL)
+
+
+function(connextdds_add_security_artifacts)
+    # Directory where the security artifacts are.
+    set(SECURITY_RESOURCES_PATH
+        ${SECURITY_RESOURCES_PATH}
```

Comment:
Do you want a list of paths or just one?

Reply:
Just one, changed to `set(SECURITY_RESOURCES_PATH "${CMAKE_MODULE_PATH}/../security")`